infra/modules/providers/azure/app-insights/README.md

```diff
@@ -1,26 +1,25 @@
 # Azure Application Insights
 
-Application Insights is an extensible Application Performance Management (APM) service for web developers on multiple platforms. Use it to monitor your live web application. It will automatically detect performance anomalies. It includes powerful analytics tools to help you diagnose issues and to understand what users actually do with your app.
+Application Insights is an extensible Application Performance Management (APM) service for web developers on multiple platforms. It is used to monitor live web applications and can automatically detect performance anomalies. It also includes powerful analytics tools to help diagnose issues and to understand what users actually do with an app.
```

Comment:
Nice corrections! Though, it might be better to place this in block quotes via the ">" syntax as this language comes directly from the App Insights documentation referenced in this Readme.